### PR TITLE
[msgpack] update version to 4.1.3

### LINKED ISSUE #28017: [msgpack] update to <4.1.3>
Library name:
msgpack
New version number:
4.1.3
Other information that may be useful (release notes, etc...)
https://github.com/msgpack/msgpack-c.git
branch: cpp_master

### PR DESCRIPTION
**Pull request by FrankXie05**
**Describe the pull request**

- #### What does your PR fix?
  Fixes https://github.com/microsoft/vcpkg/issues/28017
  Note: update `msgpack` version to 4.1.3
  No feature need to test.

### CODE REVIEW

**Comment by MonicaLiu0311** [comment]
Please resolve the conflict.

File: ports/msgpack/portfile.cmake [hunk 1]
```diff
@@ -5,35 +5,28 @@ endif()
...
+vcpkg_fixup_pkgconfig()
 
-file(REMOVE_RECURSE ${CURRENT_PACKAGES_DIR}/debug/include)
+#Clean
+file(REMOVE_RECURSE "${CURRENT_PACKAGES_DIR}/debug")
+file(REMOVE_RECURSE "${CURRENT_PACKAGES_DIR}/lib")
 
-file(COPY ${SOURCE_PATH}/COPYING DESTINATION ${CURRENT_PACKAGES_DIR}/share/msgpack)
-file(RENAME ${CURRENT_PACKAGES_DIR}/share/msgpack/COPYING ${CURRENT_PACKAGES_DIR}/share/msgpack/copyright)
+#Copyright
```
Comment (dg0yt) [inline ports/msgpack/portfile.cmake:30]: This comment is redundant.

File: ports/msgpack/vcpkg.json [hunk 2]
```diff
@@ -1,8 +1,18 @@
 {
   "name": "msgpack",
-  "version": "3.3.0",
-  "port-version": 3,
+  "version": "4.1.3",
   "description": "MessagePack is an efficient binary serialization format, which lets you exchange data among multiple languages like JSON, except that it's faster and smaller.",
   "homepage": "https://github.com/msgpack/msgpack-c",
-  "license": "BSL-1.0"
+  "license": "BSD-1-Clause",
```
Comment (dg0yt) [inline ports/msgpack/vcpkg.json:6]: No. 
https://github.com/msgpack/msgpack-c#license

**Review by dg0yt — CHANGES_REQUESTED** [review]

**Comment by JonLiu1993** [comment]
@FrankXie05, could you please fix the conflict files?

File: ports/coolprop/portfile.cmake [hunk 3]
```diff
@@ -1,10 +1,10 @@
-set(PORT_VERSION 6.1.0)
+set(PORT_VERSION 6.4.3)
```
Comment (MonicaLiu0311) [inline ports/coolprop/portfile.cmake:1]: `PORT_VERSION` is not used, please delete it.

File: ports/coolprop/portfile.cmake [hunk 4]
```diff
@@ -1,10 +1,10 @@
...
 
 vcpkg_from_github(
     OUT_SOURCE_PATH SOURCE_PATH
     REPO CoolProp/CoolProp
-    REF f5ebb4e655add4c23bb327ab5209f3dbf919bc6d # v6.4.1
-    SHA512 916d00777fe56035171ed0a6cbe09b8d4487317772802e4fe9b43f5965f3212dcb3754e18fe1db9c748a4d17facbbe6cb2244451cf5cf66334465760fc1701b7
+    REF 84897ce7a110a70ba2229e7a1f5d17ea03efd068 # v6.4.1
```
Comment (MonicaLiu0311) [inline ports/coolprop/portfile.cmake:6]: It is recommended to use `${VERSION}`, if not, please modify the comment to `# v6.4.3`.

**Review by MonicaLiu0311 — CHANGES_REQUESTED** [review]

**Review by dg0yt — DISMISSED** [review]
Okay.

Nitpick: `ports/coolprop/fix-install.patch` could be further minimized, using the new line breaks in upstream source.

**Comment by FrankXie05** [comment]
> Okay.
> 
> Nitpick: `ports/coolprop/fix-install.patch` could be further minimized, using the new line breaks in upstream source.

Done.

**Review by MonicaLiu0311 — DISMISSED** [review]

**Comment by JonLiu1993** [comment]
Please change `file(INSTALL "${SOURCE_PATH}/LICENSE" DESTINATION "${CURRENT_PACKAGES_DIR}/share/${PORT}" RENAME copyright)` to `vcpkg_install_copyright(FILE_LIST "${SOURCE_PATH}/LICENSE")`

**Review by JonLiu1993 — APPROVED** [review]

**Comment by Sharpe49** [comment]
Great... with this, it seems like you broke compatibility with applications that is using the C library, since it's no longer installed...

**Pull request merged**